<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IconMaker</class>
 <widget class="QMainWindow" name="IconMaker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Icon maker</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" rowspan="2">
     <widget class="QGraphicsView" name="graphicsView">
      <property name="minimumSize">
       <size>
        <width>256</width>
        <height>256</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>256</width>
        <height>256</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="textGroup">
      <property name="title">
       <string>Text settings</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="textLabel">
         <property name="text">
          <string>Text</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="textEdit">
         <property name="text">
          <string>Aa</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="pointSizeLabel">
         <property name="text">
          <string>Point Size</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="Slider" name="pointSizeSlider">
         <property name="minimum">
          <number>6</number>
         </property>
         <property name="maximum">
          <number>256</number>
         </property>
         <property name="value">
          <number>96</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QPushButton" name="textColourButton">
         <property name="text">
          <string>Text color</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="textXLabel">
         <property name="text">
          <string>X axis align</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSlider" name="textXSlider">
         <property name="minimum">
          <number>-64</number>
         </property>
         <property name="maximum">
          <number>64</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="textYLabel">
         <property name="text">
          <string>Y axis align</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QSlider" name="textYSlider">
         <property name="minimum">
          <number>-64</number>
         </property>
         <property name="maximum">
          <number>64</number>
         </property>
         <property name="value">
          <number>-8</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <layout class="QHBoxLayout" name="textFormatLayout">
         <item>
          <widget class="QToolButton" name="boldButton">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>B</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="italicButton">
           <property name="font">
            <font>
             <italic>true</italic>
            </font>
           </property>
           <property name="text">
            <string>I</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="underlineButton">
           <property name="font">
            <font>
             <underline>true</underline>
            </font>
           </property>
           <property name="text">
            <string>U</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QGroupBox" name="ellipseGroup">
      <property name="title">
       <string>Ellipse settings</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="ellipseSizeLayout">
         <item>
          <widget class="QLabel" name="ellipseSizeLabel">
           <property name="text">
            <string>Ellipse size (%)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Slider" name="ellipseSizelSlider">
           <property name="minimum">
            <number>10</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>90</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="penSizeLayout">
         <item>
          <widget class="QLabel" name="penSizeLabel">
           <property name="text">
            <string>Pen size (px)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Slider" name="penSizeSlider">
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>64</number>
           </property>
           <property name="value">
            <number>16</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="colourLayout">
         <item>
          <widget class="QPushButton" name="brushColourButton">
           <property name="text">
            <string>Brush colour</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="penColourButton">
           <property name="text">
            <string>Pen colour</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QPushButton" name="exportButton">
      <property name="text">
       <string>Export</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Slider</class>
   <extends>QSlider</extends>
   <header>slider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
